src/styles/global.scss

```diff
@@ -26,11 +26,6 @@
 }
 
 body {
-  padding-top: 5.25rem;
-  @extend .has-spaced-navbar-fixed-top; // padding-top: 5.25rem;
-  @include mobile {
```

Comment:
You'd better not to remove `has-spaced-navbar-fixed-top`, it is used to fix the body content when the notification banner been closed. 

You can remove the `mobile` media to fix the padding on mobile.